examples/Makefile

```diff
@@ -1,41 +0,0 @@
-JULIAHOME = $(abspath ..)
-include $(JULIAHOME)/Make.inc
-
-FLAGS = -Wall -Wno-strict-aliasing -fno-omit-frame-pointer \
-	-I$(JULIAHOME)/src -I$(JULIAHOME)/src/support -I$(build_includedir) $(CFLAGS)
-
-DEBUGFLAGS += $(FLAGS)
-SHIPFLAGS += $(FLAGS)
-JLDFLAGS += $(LDFLAGS) $(NO_WHOLE_ARCHIVE) $(call exec,$(LLVM_CONFIG) --ldflags) $(OSLIBS) $(RPATH)
-
-ifeq ($(USE_SYSTEM_LIBM),0)
-ifneq ($(UNTRUSTED_SYSTEM_LIBM),0)
-JLDFLAGS += $(WHOLE_ARCHIVE) $(build_libdir)/libopenlibm.a $(NO_WHOLE_ARCHIVE)
```

Comment:
That was added in 072a861828378 when the file was created. My guess is copy-paste or prior/local build issue. If it breaks the example we can put it back.